examples/alloc_feature.rs

```diff
@@ -21,9 +21,15 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 */
+
```

Comment:
please revert changes to this file and instead use the following more idiomatic approach:

```diff
diff --git a/Cargo.toml b/Cargo.toml
index 6dcf169..17c17f2 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -20,6 +20,10 @@ documentation = "https://docs.rs/tar-no-std"
 # required because "env_logger" uses "log" but with dependency to std..
 resolver = "2"
 
+[[example]]
+name = "alloc_feature"
+required-features = ["alloc"]
+
 [features]
 default = []
 alloc = []

```

Reply:
Done